examples/autoqasm/6_Customize_gate_calibrations.ipynb

```diff
@@ -0,0 +1,404 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "dda432a3",
+   "metadata": {},
+   "source": [
+    "# Customize gate calibrations\n",
+    "\n",
+    "Quantum hardware researches frequently require knowledge of native gate implementations on a specific devices, in the form of pulse sequences. The pulse implementation of gates is called \"gate calibrations\". In Amazon Braket, you have access to native gate calibrations provided by quantum hardware providers. Gaining access to gate calibrations empowers researchers to experiment with quantum gate designs and execute quantum programs with customized, improved gate calibrations."
```

Comment:
Don't forget to proofread
```suggestion
    "Quantum hardware researchers frequently require knowledge of native gate implementations on specific devices, in the form of pulse sequences. The pulse implementation of gates is called \"gate calibrations\". In Amazon Braket, you have access to native gate calibrations provided by quantum hardware providers. Gaining access to gate calibrations empowers researchers to experiment with quantum gate designs and execute quantum programs with customized, improved gate calibrations."
```

Reply:
This introduction makes me ask, "why do they requires this knowledge"? This is later answered. Can you restructure this paragraph so that the related information is presented more immediately? Also, I think the last sentence is actually where we state the value of this notebook overall -- so we should start with it.

You also switch from third to second person twice -- I think it's ok, but we should pick one.

Couple tips and reminders: Amazon writing style aims for simplicity, avoiding "ing" words, and avoiding imprecise terms like "frequently"